src/CustomList.php

```diff
@@ -85,6 +85,39 @@ public function offsetUnset(mixed $offset): void
         unset($this->items[$offset]);
     }
 
+    /**
+     * Concatenate to the current list the items from one or more lists
+     *
+     * @param mixed ...$lists
+     * @return void
+     * @throws CustomListTypeErrorException
+     */
+    public function concatenate(...$lists): void
```

Comment:
I propose change the name to `merge`.

Reply:
For me is not a good name because it does not delete duplicates, so its not really a merge but a concatenate